public/views/content/addStudent.html

```diff
@@ -33,7 +33,7 @@
       </div>
 
       <div class="form-group">
-          <button class="btn btn-default" ng-click = "addStudent()" type="submit">Submit</button>
+          <button class="btn btn-default" ng-disabled="!newStudent.archived" ng-click = "addStudent()" type="submit">Submit</button>
```

Comment:
not sure disabling the button is the best way to do this. Probably better to make the form fields required or something